### PR TITLE
Update playservices droid adhoc cake replacements

### PR DESCRIPTION
**Pull request by daividssilverio**
## What's this?
I've added another app entry on Firebase console for the missing adhoc configuration.
Other than that, the PR adds another string replacement for the newly added secret: `TOGGL_DROID_ADHOC_GOOGLE_SERVICES_MOBILE_SDK_APP_ID`

### Relationships
Closes #Nothing

## Why do we want this?
To test push notifications on adhoc builds.

## How is it done?
It adds missing api keys to be replaced in the `google-services.json` + the mobile sdk app id to be replaced for adhoc builds.

### Why not another way?
That's the way.

### Side effects
Droid Adhoc builds can receive pushes now.

## Review hints
Review by commit.

## :squid: Permissions
🦑 it, test it before if you can

### CODE REVIEW

**Comment by daividssilverio** [comment]
Tested and working o/

File: build.cake [hunk 1]
```diff
@@ -211,6 +212,7 @@ private TemporaryFileTransformation GetAndroidGoogleServicesTransformation()
                         .Replace("{TOGGL_PROJECT_ID}", projectId)
                         .Replace("{TOGGL_STORAGE_BUCKET}", storageBucket)
                         .Replace("{TOGGL_DROID_GOOGLE_SERVICES_MOBILE_SDK_APP_ID}", mobileSdkAppId)
+                        .Replace("{TOGGL_DROID_ADHOC_GOOGLE_SERVICES_MOBILE_SDK_APP_ID}", mobileSdkAdhocAppId)
```
Comment (simonrozsival) [inline build.cake]: I'm confused, here you call it `TOGGL_DROID_ADHOC_GOOGLE_SERVICES_MOBILE_SDK_APP_ID` and  in `google-services.json` it is called `TOGGL_DROID_GOOGLE_SERVICES_API_KEY`. How does it work?

Comment (daividssilverio) [reply build.cake]: `TOGGL_DROID_GOOGLE_SERVICES_API_KEY` is the api key, and it replaces three entries in `google-services.json`.
```
"api_key": [
        {
          "current_key": "{TOGGL_DROID_GOOGLE_SERVICES_API_KEY}"
        }
      ]
```
`TOGGL_DROID_GOOGLE_SERVICES_MOBILE_SDK_APP_ID` is the mobile sdk app id for the production apps.
`TOGGL_DROID_DEBUG_GOOGLE_SERVICES_MOBILE_SDK_APP_ID` the same, but for debug;
`TOGGL_DROID_ADHOC_GOOGLE_SERVICES_MOBILE_SDK_APP_ID` the same, but for adhoc;

Comment (daividssilverio) [reply build.cake]: Inside `google-services.json` in the `client` array, one for each `client_info` object:
```
"client": [
    {
      "client_info": {
        "mobilesdk_app_id": "{TOGGL_DROID_GOOGLE_SERVICES_MOBILE_SDK_APP_ID}",
        "android_client_info": {
          "package_name": "com.toggl.giskard"
        }
      },
...
 {
      "client_info": {
        "mobilesdk_app_id": "{TOGGL_DROID_DEBUG_GOOGLE_SERVICES_MOBILE_SDK_APP_ID}",
        "android_client_info": {
          "package_name": "com.toggl.giskard.debug"
        }
      },
...
{
      "client_info": {
        "mobilesdk_app_id": "{TOGGL_DROID_ADHOC_GOOGLE_SERVICES_MOBILE_SDK_APP_ID}",
        "android_client_info": {
          "package_name": "com.toggl.giskard.adhoc"
        }
      },
...
```

**Pull request merged**